src/main/java/vazkii/botania/common/block/BlockPlatform.java

```diff
@@ -36,14 +36,22 @@
 public class BlockPlatform extends BlockMod implements IWandable, IManaCollisionGhost, ITileEntityProvider {
 
 	public enum Variant {
-		ABSTRUSE,
-		SPECTRAL,
-		INFRANGIBLE
+		ABSTRUSE(null, false),
+		SPECTRAL(false, false),
+		INFRANGIBLE(true, true);
+
+		public final Boolean permeable;
```

Comment:
it's nonobvious at first glance what this means. Could it just be a BiPredicate<BlockPos, ISelectionContext> or something?